src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-CSharp/.template.config/template.json

```diff
@@ -3,9 +3,10 @@
   "author": "Microsoft",
   "classifications": [
     "Web",
+    "API",
```

Comment:
I'm not sure what a `classification` is - does it need to be changed?

Reply:
Is there a way I can see how these changes affect the dialog box? I thought it was like tags (for search results in the File->New dialog).

Reply:
Can we ask @seancpeters and/or @mlorbetske about it?

Reply:
@phenning can probably give the best insight into how the classifications are used by Visual Studio.

Reply:
We need to have a "web" classification for the items to show in one asp.net dialog, along with the host file.

Reply:
Okay, "API" is not required then. I have removed it from `classifications`